src/detection_win.cpp

```diff
@@ -423,6 +424,21 @@ void ExtractDeviceInfo(HDEVINFO hDevInfo, SP_DEVINFO_DATA* pspDevInfoData, TCHAR
 		// Use this to extract VID / PID
 		extractVidPid(buf, resultItem);
 	}
+
+	DWORD dwCap = 0x0;
+	if (DllSetupDiGetDeviceRegistryProperty(hDevInfo, pspDevInfoData, SPDRP_CAPABILITIES, &DataT, (PBYTE)&dwCap, 4, &nSize)) {
+		// If device has a CM_DEVCAP_UNIQUEID capability, then we assume it has a serial number
+		if ((dwCap & CM_DEVCAP_UNIQUEID) == CM_DEVCAP_UNIQUEID) {
+			if (DllSetupDiGetDeviceInstanceId(hDevInfo, pspDevInfoData, buf, buffSize, &nSize)) {
+				// extract Serial Number
+				string str = buf;
+				size_t t = str.find_last_of("\\\\");
```

Comment:
We are parsing strings like `USB\VID_046D&PID_C24E\9025267AF90009`

Why are we using a double backslash `\\\\`? It seems to work just fine with `str.find_last_of("\\");`. According to the docs, we should only need to define the backslash once, http://www.cplusplus.com/reference/string/string/find_last_of/ 🤷‍♂️ 

---

Let's rename this variable to `size_t serialNumberIndex = ...` to make this more clear

Reply:
Why double backslash? 
I have no idea :) just miswritten.